src/Sarif.Converters/ToolFormat.cs

```diff
@@ -14,8 +14,10 @@ public enum ToolFormat
         ClangAnalyzer,
         /// <summary>CppCheck's file format.</summary>
         CppCheck,
-        /// <summary>Fortify's file format.</summary>
+        /// <summary>Fortify's report file format.</summary>
         Fortify,
+        /// <summary>Fortify's FPR file format.</summary>
+        FortifyFpr,
```

Comment:
I don't object to the change, but let's do it in a separate PR. The diffs are hard to examine otherwise.

---
In reply to: [94533983](https://github.com/Microsoft/sarif-sdk/pull/554#discussion_r94533983) [](ancestors = 94533983,94533957)